packages/wonder-blocks-popover/components/popover.test.js

```diff
@@ -0,0 +1,113 @@
+// @flow
+import * as React from "react";
+
+import {mount, unmountAll} from "../../../utils/testing/mount.js";
+
+import Popover from "./popover.js";
+import PopoverContent from "./popover-content.js";
+import {PopoverContentCore} from "../index.js";
+
+describe("Popover", () => {
+    afterEach(() => {
+        unmountAll();
+    });
+
+    it("should set the anchor as the popover ref", () => {
+        // Arrange
+        const wrapper = mount(
+            <Popover
+                placement="top"
+                content={<PopoverContent title="Title" content="content" />}
+            >
+                {({open}) => (
+                    <button data-anchor onClick={open}>
+                        Open default popover
+                    </button>
+                )}
+            </Popover>,
+        );
+
+        // Act
+        const anchorElement = wrapper.find("[data-anchor]").getDOMNode();
+
+        // Assert
+        expect(wrapper.state("anchorElement")).toBe(anchorElement);
+    });
+
+    it("should hide the popover dialog by default", () => {
+        // Arrange, Act
+        const wrapper = mount(
+            <Popover
+                placement="top"
+                content={<PopoverContent title="Title" content="content" />}
+            >
+                {({open}) => (
+                    <button data-anchor onClick={open}>
+                        Open default popover
+                    </button>
+                )}
+            </Popover>,
+        );
+
+        // Assert
+        expect(wrapper.find(PopoverContent)).not.toExist();
+    });
+
+    it("should render a text-only variant", () => {
```

Comment:
The test title could be a bit more descriptive, maybe `it("should render the popover dialog after clicking the trigger", () => {...});` or something like that.